### PR TITLE
Display score on completion of quiz

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
"On completion of the quiz they will be given a total score"
- answers answered correctly out of total questions asked.

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps review please

File: src/utils_project.js [hunk 1]
```diff
@@ -154,6 +154,20 @@ go.utils_project = {
...
+    get_quiz_score: function(im) {
+        var total_questions = im.user.answers.quiz_status.questions_answered.length;
+        var correct_answers = 0;
+
+        var obj = im.user.answers.quiz_status.questions_answered;
+        for (var x in obj) {
+            if (obj[x].correct) correct_answers++;
+        }
+
+        return [correct_answers, total_questions];
```
Comment (imsickofmaps) [inline src/utils_project.js]: if you return an object you can use them by name

File: src/utils_project.js [hunk 2]
```diff
@@ -154,6 +154,20 @@ go.utils_project = {
             });
     },
 
+    // returns a 2-element array; first value represents the number of correct
+    // answers, and second value the total number of questions asked
+    get_quiz_score: function(im) {
```
Comment (imsickofmaps) [inline src/utils_project.js]: lets also return the percentage from this and make it `get_quiz_summary`

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps 2nd pass review please

**Comment by imsickofmaps** [comment]
👍

**Pull request merged**